### DISCUSSION TITLE
Permissions for specific project in one customer

### DISCUSSION THREAD

**Discussion by hernanPSS** [kimai/kimai#4718] [Questions & Answers] [answered]
I'm trying to let a team see only 1 of 3 projects of the same customer....
if I give access to the team to the customer, the user can see all the projects (I only checked the customer and 1 project)
if I uncheck the customer the user of the team can see nothing of that customer....

any idea?

**Comment by hernanPSS** [comment]
![kimai](https://github.com/kimai/kimai/assets/73197539/8a1cb740-2fa8-4438-9480-10029608eb1e)

**Reply by kevinpapst** [reply, marked as answer]
you need to limit the other projects by linking other teams to those projects
